src/components/UseOfFunds/FundsDistributionAccordion.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+	<div class="funds-distribution-accordion">
+		<div v-for="( fundsItem, idx ) in applicationOfFundsData" :key="idx" :class="[
+			'funds-distribution-accordion-item',
+			'funds-distribution-accordion-item-' + fundsItem.id,
+			{ 'active' : currentActiveItems.includes( String( idx ) ) }
+		]">
+			<button class="funds-distribution-accordion-item-title" @click="toggleActive( String( idx ) )">
```

Comment:
Typescript throws issues on compile when you try use idx as a number, but is fine when we cast it to a string. Guessing it gets changed somewhere along the build pipeline.

Reply:
Yeah, but somewhere along the pipeline it must get transposed to a string.